uberfire-structure/uberfire-structure-api/src/main/java/org/guvnor/structure/repositories/changerequest/portable/ChangeRequestCommit.java

```diff
@@ -0,0 +1,27 @@
+package org.guvnor.structure.repositories.changerequest.portable;
+
+import org.jboss.errai.common.client.api.annotations.Portable;
+
+@Portable
+public class ChangeRequestCommit {
+
+    private String id;
+    private String message;
+
+    public ChangeRequestCommit() {
```

Comment:
Consider removing this empty constructor.

Reply:
Done